Common/src/main/java/com/tabnineCommon/chat/actions/common/ChatActionCommunicator.kt

```diff
@@ -0,0 +1,54 @@
+package com.tabnineCommon.chat.actions.common
```

Comment:
extracted logic from `AskChatAction`, to be used in `TabnineQuickFixAction` (and potentially future actions) as well.